src/main/java/com/google/devtools/build/lib/rules/platform/Platform.java

```diff
@@ -42,6 +45,18 @@ public ConfiguredTarget create(RuleContext ruleContext)
 
     PlatformInfo.Builder platformBuilder = PlatformInfo.builder().setLabel(ruleContext.getLabel());
 
+    List<PlatformInfo> parentPlatforms =
+        Lists.newArrayList(
+            PlatformProviderUtils.platforms(
+                ruleContext.getPrerequisites(PlatformRule.PARENTS_PLATFORM_ATTR, Mode.DONT_CHECK)));
+
+    if (parentPlatforms.size() > 1) {
+      throw ruleContext.throwWithAttributeError(
+          PlatformRule.PARENTS_PLATFORM_ATTR, "parents attribute must have a single value");
```

Comment:
Optional nit: replace "parents" with PlatformRule.PARENTS_PLATFORM_ATTR to further consolidate the source of truth?

Reply:
Done.